src/components/OrderList/OrderList.js

```diff
@@ -66,8 +58,16 @@ function OrderList({ filter }) {
             <td>
               {order.buyer.firstName} {order.buyer.lastName}
             </td>
-            <td>Rose Necklace</td>
-            <td>{order.quantity}</td>
+            <td>
+              {order.items.map((item) => (
+                <>
+                  <label className="text-capitalize mb-0">
+                    {item.product.name} (x{item.quantity})
+                  </label>{" "}
+                  <br />
```

Comment:
- Try to avoid using `<br />`s. Use it as a last resort. 
- Leave the capitalization as it is. If the merchant named their product neckLace we can't display it as -> NeckLace. _(Maybe that is part of their branding)_. 
- The `{" "}` is unnecessary if you use a block level element.